CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: Exposed Metroid Queen-Locked doors for Door Lock Rando.
 - Added: Exposed Open Transitions for Door Lock Rando. Shuffling these in, will place Doors on all 4-Tile high transitions.
 - Added: Transport Pipe randomizer. If enabled, it will change where Transport Pipes lead to.
+- The following sprites have been added: Spider Ball for Prime 1, Missile Launcher, Speed Booster Upgrades and Super Missile Launcher for Dread, Baby Metroid, Beam Burst, Lightning Armor, Phase Drift, Scan Pulse and Spider Ball and Grapple Beam for Metroid Samus Returns.
```

Comment:
Probably don't need to mention MSR just yet

Reply:
fair enough.